### PR TITLE
Keep supported CentOS versions in plugins/centos.conf

### PR DESCRIPTION
**Pull request by michalfabik**
See commit.

Signed-off-by: Michal Fabik <mfabik@redhat.com>

### CODE REVIEW

File: src/pyfaf/opsys/centos.py [hunk 1]
```diff
@@ -187,7 +193,20 @@ def save_ureport(self, db, db_report, ureport, packages, flush=False, count=1):
...
+
+        for release in re.split(r"\s*[,;]\s*|\s+", self.inactive_releases):
+            result[release] = {
+                "status": "EOL",
+                "shortname": "centos-{0}".format(release)
+            }
+        for release in re.split(r"\s*[,;]\s*|\s+", self.active_releases):
+            result[release] = {
+                "status": "ACTIVE",
+                "shortname": "centos-{0}".format(release)
```
Comment (xsuchy) [inline src/pyfaf/opsys/centos.py]: why the shortname? it was not previously in that dictionary.

Comment (michalfabik) [reply src/pyfaf/opsys/centos.py]: I suppose i just wanted to keep it consistent with the [way its done for Fedora](https://github.com/abrt/faf/blob/f18b566afa2abfbd61d9278a44b6488b36d10430/src/pyfaf/opsys/fedora.py#L229), but you're right that it doesn't seem to be needed for either of the systems at the moment.

Comment (ernestask) [reply src/pyfaf/opsys/centos.py]: I think it’s a derelict from the PkgDB days, but even then it’s not quite clear what it was for.

Comment (michalfabik) [reply src/pyfaf/opsys/centos.py]: I see. Should I remove it from the Fedora part as well while I'm at it?

Comment (ernestask) [reply src/pyfaf/opsys/centos.py]: I can’t see any harm coming from doing it, so why not.

File: config/plugins/centos.conf [hunk 2]
```diff
@@ -1,3 +1,5 @@
 [CentOS]
 base-repo-url=http://vault.centos.org/centos/$releasever/os/Source/
 updates-repo-url=http://vault.centos.org/centos/$releasever/updates/Source/
+inactive-releases=5, 6
```
Comment (mkutlak) [inline config/plugins/centos.conf]: With this the 5 and 6 are added to FAF (even if as EOL) and we start to process uReports from them. And we not really want that. _(If uReport comes from unknown OpSys, it is rejected and moved to `deferred` dir.)_


So I think the `incactive-releases` should be empty by default.

Comment (michalfabik) [reply config/plugins/centos.conf]: removed

Comment (michalfabik) [reply config/plugins/centos.conf]: removed

Comment (ernestask) [reply config/plugins/centos.conf]: removed

**Pull request merged**